src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -1596,13 +1625,6 @@ public void handle(final Signal signal) {
         }
     }
 
-    /**
-     * @return Obtain the model of CPU on Linux or the os.arch on other OSes.
-     */
-    public static String getCpuClass() {
```

Comment:
possibly a breaking API change!

Reply:
This static method was moved to before the non static members by the IDE